app/templates/questionnaire.html

```diff
@@ -63,7 +63,7 @@ <h1 class="panel__title venus">This page has {{all_errors|length}} errors</h1>
     {% endif %}
 
     <form action="{{form_action|default('')}}" class="form qa-questionnaire-form" role="form" method="POST" novalidate>
-
+      <input type="submit" style="display: none" name="action[save_continue]" />
```

Comment:
I don't think we should be introducing this, especially to all questionnaires.

The non-js version (which nobody will use) behaves exactly as it's supposed to, this adds the risk of unwanted side-effects.